frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,54 @@
+.input {
+  padding: 20px 24px;
+  font-family: "Work Sans", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  color: #000;
+  border: 1px solid #9f18d9;
+  border-radius: 10px;
+  box-shadow: 0 4px 5px #9f18d9 0.5;
+}
+
+.input:active {
+  color: #000;
+  outline-style: none;
```

Comment:
Removing `outline` without adding back styles in a different way to signal these state changes is a really bad practice for accessibility.

Let's keep the browser defaults until we have a design for them. Or, let's not add any styles that we still don't have the design for.

Reply:
@mirhamasala @RicardoVaz1 we need to ask Hanna what the focused/active state should look like

Reply:
Ok, I'll remove it for now, and I'll ask Hannah the designs for focused/active state